src/extension.ts

```diff
@@ -75,6 +75,17 @@ export function activate(context: ExtensionContext) {
 
   function getSettings(): JSON {
     let configXML = workspace.getConfiguration();
+
+    let autoCloseTags = configXML.get("xml.completion.autoCloseTags");
```

Comment:
this should only happen on configuration change on the client side.
there's no need to send that config to the server